src/main/java/es/thalesalv/chatrpg/adapters/data/entity/UserDefinitionsEntity.java

```diff
@@ -0,0 +1,21 @@
+package es.thalesalv.chatrpg.adapters.data.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "user_definitions")
+public class UserDefinitionsEntity {
+
+    @Id
```

Comment:
Yes, it's the object that will be used for both NAI/GAI/OAI expansions and Discord guild segregation. Since you was going to work on one thing and I was going to work on another, I figured it would be easier to have the base for both already implemented.